internal/server/assign_task.go

```diff
@@ -127,6 +133,12 @@ func AssignTask(client AssignTaskClient, tmpl template.Template) Handler {
 				delete(data.Error.Field, "assigneeId")
 			} else if err != nil {
 				return err
+			} else if lpa != nil && lpa.CaseType == "DIGITAL_LPA" {
```

Comment:
```suggestion
			} else if task.CaseItems[0].CaseType == "DIGITAL_LPA" {
```
I'm wondering if you could simplify the logic to this, as surely theres not likely to ever be a task that does not have a case attached to it? And if so then its a bad task object

Reply:
This saves us having to keep an array of all the tasks (more than one can be passed to this handler). Instead, we just store a reference to the first case associated with the first task which has cases.

While you're correct that a task without cases would be bad, as far as the handler is concerned, a task doesn't need to have a case for this to work: a task without a case can be assigned to a different user. We only care about cases when we want to redirect for digital LPAs.